File: app/src/main/java/fr/everydaysapps/marvelsuperheroes/models/Hero.java
package fr.everydaysapps.marvelsuperheroes.models;

import com.google.gson.annotations.Expose;

import java.io.Serializable;

/**
 * Created by ygharsallah on 30/03/2017.
 */

public class Hero  implements Serializable{

    @Expose
    private String image;
    @Expose
    private String title;
    @Expose
    private String intro;
    @Expose
    private String year;
    @Expose
    private String text;
    @Expose
    private String color;

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getIntro() {
        return intro;
    }

    public void setIntro(String intro) {
        this.intro = intro;
    }

    public String getYear() {
        return year;
    }

    public void setYear(String year) {
        this.year = year;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

}

File: app/src/main/java/fr/everydaysapps/marvelsuperheroes/data/Hero.kt
package fr.everydaysapps.marvelsuperheroes.data

import com.google.gson.annotations.SerializedName

data class Hero(
    @SerializedName("name") val name: String,
    @SerializedName("photo") val photo: String,
    @SerializedName("realName") val realName: String,
    @SerializedName("height") val height: String,
    @SerializedName("power") val power: String,
    @SerializedName("abilities") val abilities: String,
    @SerializedName("groups") val groups: String
)
